### DISCUSSION TITLE
add sort-by-selection feature

### DISCUSSION THREAD

**Discussion by vincentaxhe** [helix-editor/helix#11785] [Ideas]
after selection, :sort-by-selection sort lines by selection of each line.
for example
```
acommand = ls
bcommand = cd
```
if want to sort by 'ls' and 'cd', then I can select '= .' to filter them, and use :sort-by-selection to sort whole lines, sort only selection is rare, it mess up the lines.
or Helix has already implement it.

**Comment by wldmr** [comment]
You can pipe the selection to the external `sort` command:

```
$ sort --help
[…]
  -k, --key=KEYDEF          sort via a key; KEYDEF gives location and type
[…]
  -t, --field-separator=SEP  use SEP instead of non-blank to blank transition
[…]
KEYDEF is F[.C][OPTS][,F[.C][OPTS]] for start and stop position, where F is a
field number and C a character position in the field; both are origin 1, and
the stop position defaults to the line's end.
```

So for your example that would mean `| sort -t= -k2` does the job.
